rules/codelingo/effective-go/good-package-name/codelingo.yaml

```diff
@@ -23,17 +23,27 @@ tenets:
       codelingo/docs:
         title: Good Package Name
         body: |
-          It's helpful if everyone using the package can use the same name 
-          to refer to its contents, which implies that the package name should 
-          be good: short, concise, evocative. By convention, packages are 
-          given lower case, single-word names; there should be no need for 
-          underscores or mixedCaps. Err on the side of brevity, since everyone 
-          using your package will be typing that name. And don't worry about 
+          A package name should be short and concise. Everyone using the
+          package should use the same name to refer to its contents.
+          By convention, packages are given lower case, single-word names. 
+          There is no need for underscores or mixedCaps. Do not worry about 
           collisions a priori. The package name is only the default name for 
```

Comment:
Perhaps I'm just not a big enough Latin-nerd, but I reckon we could make the original text simpler. 

What do you reckon about this instead?
> Do not worry about potential naming collisions.